modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/components/TableKeyDetails/index.js

```diff
@@ -29,29 +29,29 @@ const SUBSCRIPTION_IMAGE_FILE = 'dxp_icon.svg';
 const NO_EXPIRATION_DATE = 100;
 
 const TableKeyDetails = ({
-	activationKeys,
 	assetsPath,
+	currentActivationKey,
 	setValueToCopyToClipboard,
 }) => {
 	const [actionToCopy, setActionToCopy] = useState('');
-	const instanceSizeFormated = activationKeys.sizing?.slice(7, 8);
+	const instanceSizeFormated = currentActivationKey.sizing?.slice(7, 8);
```

Comment:
I think would be better explicit the string format here, what is being removed?
obs: I am not sure about the text if is 'Sizing' or 'sizing', you can verity it before change 😉

```suggestion
	const instanceSizeFormated = currentActivationKey.sizing?.replace('Sizing ', '');
```

Reply:
You can use the function that you create on path: modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/routes/customer-portal/containers/DXPActivationKeysTable/utils/getInstanceSize.js